public/app/plugins/panel/news/models.gen.ts

```diff
@@ -11,6 +11,6 @@ export interface PanelOptions {
   showImage?: boolean;
 }
 
-export const defaultPanelOptions: PanelOptions = {
+export const defaultPanelOptions: Partial<PanelOptions> = {
```

Comment:
does it need to be partial?

Reply:
Plausibly not? But this struck me as a cuttable corner, so i cut it: grafana/cuetsy#54